asreview/webapp/src/ProjectComponents/AnalyticsComponents/LabelingFrequency.js

```diff
@@ -246,7 +160,56 @@ const LabelingFrequency = ({ genericDataQuery, progressQuery }) => {
           </>
         )}
       </CardContent>
-    </StyledCard>
+      <Popover
+        id="info-popover"
+        open={popoverOpen}
+        anchorEl={anchorEl}
+        onClose={handlePopoverClose}
+        PaperProps={{
+          sx: {
+            p: 3,
+            maxWidth: 300,
+            borderRadius: 2,
+            boxShadow: 3,
+            bgcolor: theme.palette.background.paper,
+            color: theme.palette.text.primary,
+          },
+        }}
+      >
```

Comment:
Popovers work nicely!

Also here, let's minimize styling. We move this to the custom theme.